05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,51 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+#bothboxes img { /*Container with both boxes*/
+  width:500px;
+  float:left;
+  display: inline-block;
+  position:relative;
 }
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+#graybox { /*Gray box on the right*/
+  height:400px;
+  width:500px;
+  background-color:#F0F0F0;
+  position:fixed;
```

Comment:
setting a width and a max-width for the same element is a bit redundant and sometimes can mess up your layout.